### ISSUE TITLE
OZP Deployment via Vagrant Yields Errors

### ISSUE THREAD

**Issue by javaTDI** [ozone-development/dev-tools#12]
I posted to the user group (https://groups.google.com/forum/#!topic/ozoneplatform-users/9NDXL_qqXck) and sent an email (30 minutes ago, not rushing him, just posting here to cover all grounds for visibility)

I've attempted twice (actual more than that, but for brevity) to deploy OZP in 
a single VM environment via vagrant.
    Host (physical machine) being Windows 7
        Cygwin installed with ssh and rsync
        Executed "vagrant up" from within dev-tools\vagrant\centos6.6\build-and-deploy-box directory
    Mac OS El Capitan host (physical machine)
        CentOS7 VM with Virtual Box and Vagrant installed (yes, a VM within a VM to rule out the issue was NOT just a Windows environment, didn't want to deploy this directly on my Mac)
        Executed "vagrant up" from within dev-tools/vagrant/centos6.6/build-and-deploy-box directory

Both setups yield the same result, HUD, Webtop and Center does not load 
properly (as if it is waiting on another service, perhaps Marketplace which 
seems to be MIA).

Since vagrant is managing the deployment of the CentOS 6.6 VM housing OZP and it pulls the dependencies needed, perhaps there is a script error (in authorship or perhaps outdated), just guessing without dissecting the script.  Question, I noticed that there is a new directory within dev-tools\vagrant: "new-backend", is this a complete OZP environment error free (at least from just a basic deployment and successful initialization (starting of services))?  If not what are the steps to correct the errors below?

I've attached the entire log, vagrant.txt as well as an abbreviated one (just highlighting the actual errors from the log), highlightedErrorsFromVagrantLog.txt

[vagrant.txt](https://github.com/ozone-development/dev-tools/files/64543/vagrant.txt)
[highlightedErrorsFromVagrantLog.txt](https://github.com/ozone-development/dev-tools/files/64544/highlightedErrorsFromVagrantLog.txt)

**Comment by tonyalletag** [comment]
did fixing the gvmtool errors resolve this issue?

**Comment by javaTDI** [comment]
Tony, the error that user jkins pointed out was one of the errors I highlighted in the logs I sent you back in November/December.  Which VM was your script updates verified against, CentOS vagrant or the Ubuntu vagrant?  I am asking the question without pulling anything from GitHub.  Thanks.

**Comment by tonyalletag** [comment]
The updates were on the the CentOS vagrant script.  We have had a few people run the script successfully.  Please submit a new issue if you have any errors.
